test/vaadin-radio-group.html

```diff
@@ -548,6 +548,47 @@
 
           expect(spy).to.not.be.called;
         });
+    
+        it('should value on the vaadin-radio-group is updated at the time when change is fired by clicking mouse', done => {
+          // Mouse
+          vaadinRadioButtonGroup.addEventListener('change', e => {
+            expect(vaadinRadioButtonGroup.value).to.equal(vaadinRadioButtonList[2].value);
+            done();
+          });
+          vaadinRadioButtonList[2].click();
+        });
+
+        it('should change event on radio-group includes updated value after clicking mouse', done => {
+          vaadinRadioButtonGroup.addEventListener('change', e => {
+            expect(e.target.textContent).to.equal(vaadinRadioButtonList[2].textContent);
+            expect(e.currentTarget.value).to.equal(vaadinRadioButtonList[2].value);
+            done();
+          });
+          vaadinRadioButtonList[2].click();
+        });
+
+        it('should value on the vaadin-radio-group is updated at the time when change is fired by pressing down', (done) => {
```

Comment:
Please replace `(done)` with `done` as used elsewhere in the tests (no parentheses).